rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/LicenseController.java

```diff
@@ -525,4 +525,34 @@ public ResponseEntity<RequestStatus> createLicenseType(
         HttpStatus status = HttpStatus.OK;
         return new ResponseEntity<>(requestStatus, status);
     }
+
+    @Operation(
+            summary = "List all of the service's licenses.",
+            description = "List all of the service's licenses.",
+            tags = {"Licenses"}
+    )
+    @RequestMapping(value = LICENSES_URL+"/quickFilter", method = RequestMethod.GET)
```

Comment:
Instead of creating new endpoint, I'll update the existing `getLicenses()` with a new parameter like `?search`. Simply use the filtered results if this parameter exists, else get all licenses as usual.